### PR TITLE
Tcs 159 add segmentid to access tables

### PR DESCRIPTION
**Pull request by esweeney97**
Adds segment information to a proc's access table which then gets published to the gateway access table.

### CODE REVIEW

**Comment by esweeney97** [comment]
The tailreader just loads the access table that gets saved down by the tailer so the tailreader will already have the new column

**Comment by ThomasGillespie** [comment]
oh, yeah that makes sense

**Review by ThomasGillespie — APPROVED** [review]

**Review by jonathonmcmurray — APPROVED** [review]

File: code/rdb/datastripe.q [hunk 1]
```diff
@@ -27,7 +27,7 @@ initdatastripe:{
     endofperiod::.rdb.datastripeendofperiod;
     .rdb.tablekeycols:.ds.loadtablekeycols[];
     t:tables[`.] except .rdb.ignorelist;
-    .ds.access:([table:t] start:.ds.getstarttime each t; end:0Np ; stptime:0Np ; keycol:`sym^.rdb.tablekeycols[t]);
+    .ds.access:([table:t] start:.ds.getstarttime each t; end:0Np ; stptime:0Np ; keycol:`sym^.rdb.tablekeycols[t]; segment:first .ds.segmentid);
```
Comment (drgdavies) [inline code/rdb/datastripe.q:30]: Any reason why this is first?

Comment (dmitchard) [reply code/rdb/datastripe.q:30]: .ds.segmentid is a one item list, not really sure why, but I imagine this is why Ethan is using first to get the value

Comment (esweeney97) [reply code/rdb/datastripe.q:30]: .ds.segmentid is a list of one symbol, was throwing a `length` error when loading the process code

**Pull request merged**